CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 ### Changed
 
 - Fixed no-unused-expressions
+- Removed `Microscopy` from export names
```

Comment:
The next release is 0.0.6 now.

Reply:
Cool: Just pushed an update. (I've been putting "in progress" on the latest entry in the vitessce changelog, and stubbing out a new empty section when I make a release, so I remember to put incoming notes in the right place.)